src/containerapp/HISTORY.rst

```diff
@@ -17,6 +17,7 @@ Upcoming
 * Add 'az containerapp ingress cors' for CORS support
 * 'az container app env create/update': support --enable-mtls parameter
 * 'az containerapp up': fix issue where --repo throws KeyError
+* 'az containerapp create/update': --termination-grace-period support custom termination grace period
```

Comment:
Please do not add the history note into a extension version which had already been released.

Reply:
this means support a new property when az containerapp create/update, maybe I can change the description better

Reply:
If you want to release a new extension version for this PR, you need to upgrade the version defined in `HISTORY.rst` and `setup.py`. 
If you don't want to release a new extension version, you can put the history note under `Upcoming`

Reply:
I rebase main branch to fix a conflict just now

Reply:
it seems us team released a new version last night... Have updated it